docs/what-is-rocfft.rst

```diff
@@ -22,6 +22,11 @@ The rocFFT library provides a fast and accurate platform for calculating discret
 * Real and complex FFTs
 * Arbitrary lengths, with optimizations for combinations of powers of 2, 3, 5, 7, 11, 13, and 17
 
+rocFFT also has experimental support for:
+
+* Distributing transforms across multiple GPU devices in a single process
```

Comment:
Here we are introducing the idea of distributed transforms, which are new and experimental, but we have not actually discussed in-place transforms, which have existed but are not discussed here.  Maybe we should briefly discuss the difference between in-place and distributed transforms prior to this?